### PR TITLE
[Player][ACR-1219] Add 360 video feature key

### PR DESCRIPTION
**Pull request by eunhaechoi**
This modification is for ACR-1219.

### CODE REVIEW

**Comment by journey2w** [comment]
@annie-samsung 
Please review english sentences.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 1]
```diff
@@ -347,6 +347,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/media.video_recording`    | Specify this key, if the application requires the video recording feature. | 2.3 |
 | `http://tizen.org/feature/microphone`    | Specify this key, if the application requires a microphone. | 2.2.1 |
 | `http://tizen.org/feature/multimedia.transcoder` | Specify this key, if the application requires the multimedia transcoder feature. | 2.3   |
+| `http://tizen.org/feature/multimedia.player.spherical_video` | Specify this key, if the application requires the 360-degree video playback feature. | 5.0   |
 | `http://tizen.org/feature/multi_point_touch.pinch_zoom` | Specify this key, if the application requires a pinch-zoom gesture feature. | 2.2.1 |
 | `http://tizen.org/feature/multi_point_touch.point_count` | Specify this key with a specific number (`int` type), if the application requires the minimum of specified number of simultaneous touches in a multi-point touch. | 2.2.1 |
 | `http://tizen.org/feature/network.bluetooth` | Specify this key, if the application requires the Bluetooth feature. | 2.2.1 |
```
Comment (divyk) [inline docs/application/native/tutorials/details/app-filtering.md]: Specify this key when the application requires a Bluetooth feature.

File: docs/application/native/guides/device/system.md [hunk 2]
```diff
@@ -311,6 +311,7 @@ The following table lists the multimedia feature keys.
 | Key                                      | Type   | Description                              |
 |------------------------------------------|--------|------------------------------------------|
 | `http://tizen.org/feature/multimedia.transcoder` | `bool` | The platform returns `true` for this key, if the device supports the transcoder. |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/guides/device/system.md]: Hello, divyk.

The reviewed result by you changed the order of 2 sentences. 
However this re-ordering breaks the consistency with other lots of similar sentences.

So, how about keeping the current order as it is?

File: docs/application/native/guides/device/system.md [hunk 3]
```diff
@@ -311,6 +311,7 @@ The following table lists the multimedia feature keys.
...
+| `http://tizen.org/feature/multimedia.player.spherical_video` | `bool` | The platform returns `true` for this key, if the device supports the 360-degree video playback feature. |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/guides/device/system.md]: Hello, divyk.

The reviewed result by you changed the order of 2 sentences. 
However this re-ordering breaks the consistency with other lots of similar sentences.

So, how about keeping the current order as it is?

File: docs/application/native/guides/device/system.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/guides/device/system.md]: @mijin85cho , 
Ok I agree.
Thanks.

File: docs/application/native/guides/device/system.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/guides/device/system.md]: @mijin85cho , 
Ok I agree.
Thanks.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 4]
```diff
@@ -347,6 +347,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/media.video_recording`    | Specify this key, if the application requires the video recording feature. | 2.3 |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: Hello, divyk.

The reviewed result by you changed the starting of the sentence to `Specify this key when the application requires...`
However this re-ordering breaks the consistency with other lots of similar sentences.

So, how about keeping the current `Specify this key, if the application requires...` as it is?

Also, I have a question : Is it ok to uses the singular form for the feature here?

File: docs/application/native/tutorials/details/app-filtering.md [hunk 5]
```diff
@@ -347,6 +347,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
...
 | `http://tizen.org/feature/microphone`    | Specify this key, if the application requires a microphone. | 2.2.1 |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: Same query with above one.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 6]
```diff
@@ -347,6 +347,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
...
 | `http://tizen.org/feature/multimedia.transcoder` | Specify this key, if the application requires the multimedia transcoder feature. | 2.3   |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: Same query with above one.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 7]
```diff
@@ -347,6 +347,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
...
+| `http://tizen.org/feature/multimedia.player.spherical_video` | Specify this key, if the application requires the 360-degree video playback feature. | 5.0   |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: Same query with above one.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 8]
```diff
@@ -347,6 +347,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
...
 | `http://tizen.org/feature/multi_point_touch.pinch_zoom` | Specify this key, if the application requires a pinch-zoom gesture feature. | 2.2.1 |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: Same query with above one.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 9]
```diff
@@ -347,6 +347,7 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
...
 | `http://tizen.org/feature/multi_point_touch.point_count` | Specify this key with a specific number (`int` type), if the application requires the minimum of specified number of simultaneous touches in a multi-point touch. | 2.2.1 |
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: Same query with above one.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 1 above)
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: Same query with above one.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: @mijin85cho ,
Ok I agree.
Yeah, in this case it is talking about the video recording feature, no other features are mentioned here, hence singular suits. This is based on my limited knowledge.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 1 above)
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree.

**Comment by mijin85cho** [comment]
Hello, eunhaechoi

There is one review comment to be checked & updated by you. Would you please check & update it?

@divyk 
When eunhaechoi update this PR, please check the updated version and approve it if you are ok with the update.

**Comment by mijin85cho** [comment]
Hello, @divyk

There are some updates for this PR after your review.
Would you please re-check the latest update?

Regards

File: docs/application/native/guides/device/system.md [hunk 10]
```diff
@@ -311,6 +311,7 @@ The following table lists the multimedia feature keys.
...
 
 The following table lists the multi-point touch feature keys.
```
[the remark this answers is not in the document]
Comment (divyk) [reply docs/application/native/guides/device/system.md]: Ok, thanks for the clarification.

File: docs/application/native/guides/multimedia/media-playback.md [hunk 11]
```diff
@@ -13,7 +13,9 @@ The main media playback features are:
 
   Enables you to play [video](#video).
 
-- Using the player [features](#features)Enables you manage the player and control the volume, sound mode, display, stream info, and audio effects.
+- Using the player [features](#features)
+
+  Enables you manage the player and control the volume, sound mode, display, stream info, and audio effects.
```
Comment (divyk) [inline docs/application/native/guides/multimedia/media-playback.md]: Enables you to manage..

File: docs/application/native/guides/multimedia/media-playback.md [hunk 12]
```diff
@@ -126,6 +124,10 @@ The Player sub-APIs offer the following features:
 
   After changing to the READY state, the subtitles can be read.
 
+- Media Streams
+
+  Use the media stream API(in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__SUBTITLE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__SUBTITLE__MODULE.html) applications) to play and control the external demuxed stream.
```
Comment (divyk) [inline docs/application/native/guides/multimedia/media-playback.md]: a space can be given between API and (in mobile

File: docs/application/native/guides/multimedia/media-playback.md (hunk 11 above)
Comment (eunhaechoi) [reply docs/application/native/guides/multimedia/media-playback.md]: I will update at next patch. Thank you.

File: docs/application/native/guides/multimedia/media-playback.md (hunk 12 above)
Comment (eunhaechoi) [reply docs/application/native/guides/multimedia/media-playback.md]: I will update at next patch. Thank you.

Comment (eunhaechoi) [reply docs/application/native/guides/multimedia/media-playback.md]: it is updated.

File: docs/application/native/guides/multimedia/media-playback.md (hunk 11 above)
Comment (eunhaechoi) [reply docs/application/native/guides/multimedia/media-playback.md]: it is updated.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: One more question for this update.

It looks like http://tizen.org/feature/multimedia.player.spherical_video is supported by both mobile and wearable.

However, it is added on Table for  "Available wearable native requirements". 

Don't we need to add it to table for mobile, too?

[the remark this answers is not in the document]
Comment (eunhaechoi) [reply docs/application/native/tutorials/details/app-filtering.md]: I think I did not catch that the feature table is seperated.
As your comment, it have to be added at mobile too.
I added new patch. Thank you.

**Comment by mijin85cho** [comment]
As other parts are reviewed by divyk,
and as the added sentence is copied one from existing, I consider this PR is reviewed & approved.

Thanks for the review & update.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**